README.md

```diff
@@ -1,6 +1,12 @@
 # aiidalab-dev
 Development utilities for AiiDAlab
 
+This package helps with the development of `aiidalab` and `aiidalab-home`
+packages by installing them in a sandboxed development environment in `~/local/`.
+
+**NOTE**: This package is meant to be run inside the AiiDAlab Docker stack
```

Comment:
Just for the `aiidalab-home`, can you add a note for the problem I report? Since it uses the soft link I think user desire to know how it works differently.

Reply:
I am not sure what to write here. The issue you reported only happened because you've already altered your default environment. Whereas users of this are supposed to use the `develop-aiidalab` to achieve essentially the same effect as what you did manually. We can improve the message that you got to make it clearer, but I'd do that in a separate PR.

Reply:
Sure! I agree, like printing out where is the "unknown" path linked.